AirCasting/APICommunicator/CreateSessionAPI.swift

```diff
@@ -130,3 +132,33 @@ final class CreateSessionAPIService {
         }
     }
 }
+
+protocol BaseURLProvider {
+    var baseAppURL: URL { get set }
+    var authorizationURL: URL { get }
+}
+
+extension BaseURLProvider {
+    var authorizationURL: URL {
+        URL(string: "http://aircasting.org/api")!
+    }
+}
+
+class UserDefaultsBaseURLProvider: BaseURLProvider {
+    var baseAppURL: URL {
+        set {
+            userDefaults.set(newValue, forKey: "baseURL")
+        }
+        get {
+            userDefaults.url(forKey: "baseURL") ?? URL(string: "http://aircasting.org/api")!
+        }
+    }
+    private let userDefaults: UserDefaults
+    init(userDefaults: UserDefaults = .standard) {
+        self.userDefaults = userDefaults
+    }
+}
+
+struct DummyURLProvider: BaseURLProvider {
```

Comment:
can we wrap this inside 
```Swift
#if DEBUG
#endif
```
??
And perhaps also add it to some other file (maybe the same as the `protocol`? I feel like this dummy implementation doesn't deserve its own file)